OpenRA.Mods.Common/Traits/Player/DummyBot.cs

```diff
@@ -17,8 +17,9 @@ namespace OpenRA.Mods.Common.Traits
 	[TraitLocation(SystemActors.Player)]
 	public sealed class DummyBotInfo : TraitInfo, IBotInfo
 	{
+		[FluentReference]
```

Comment:
This probably also wants a `FieldLoader.Require`?

Reply:
sure, the former behaviour already triggered lint tests though